code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -204,6 +204,26 @@
 		if("harm")
 			wuv(-1, M)
 
+/mob/living/simple_animal/pet/cat/attack_alien(mob/living/carbon/alien/humanoid/M)	//what the xeno was after all along in Alien
+	//. = ..()	//I assume this is needed

```

Comment:
make it ```else return ..()``` for the below if.
what you've done is prevent aliens from attacking cats, ever, regardless of intentions.